scripts/comparison/compare_results.py

```diff
@@ -6,17 +6,20 @@
 import xml.etree.ElementTree as ET
 from html_report import htmlReportSection, htmlReport
 import glob
+import configparser
 
 
 def getParameters(argv):
```

Comment:
use pep8/flake8 and fix the issues

Reply:
python methods are not camelCase - they are camel_case and class names should be ClassName

Reply:
Fix all the occurrences of these issues